pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1520,6 +1520,10 @@ class Timedelta(_Timedelta):
         div = other // self
         return div, other - div * self
 
+    def total_seconds(self):
```

Comment:
can you add a full-docstring, e.g. need examples, wouldn't hurt to reference the python timedelta.total_seconds method as well.

Reply:
Done